src/OmniSharp.MSBuild/MSBuildProjectSystem.cs

```diff
@@ -308,7 +308,7 @@ private string GetSdksPath(string projectFilePath)
         {
             var info = _dotNetCli.GetInfo(Path.GetDirectoryName(projectFilePath));
 
-            if (info.IsEmpty && !string.IsNullOrWhiteSpace(info.BasePath))
+            if (info.IsEmpty && string.IsNullOrWhiteSpace(info.BasePath))
```

Comment:
I think you want `||` here. Note that `info.BasePath` will always be null if `info.IsEmpty` is true (see [`DotNetInfo.cs`](https://github.com/OmniSharp/omnisharp-roslyn/blob/dev/src/OmniSharp.Abstractions/Services/DotNetInfo.cs))